lib/metric-config-parser/metric_config_parser/tests/sql/test_generate_data_source_with_join.expected.sql

```diff
@@ -28,4 +28,4 @@
     ON 
     joined_baseline.client_id = events.client_id
     
-            )
+                )
```

Comment:
We should probably auto-format generated SQL so that tweaking the templates doesn't break all the tests due to changes in indentation. I'll open a ticket